.eslintrc.js

```diff
@@ -0,0 +1,241 @@
+'use strict';
+
+const Module = require('module');
+const path = require('path');
+
+const NodePlugin = require('./tools/node_modules/eslint-plugin-node');
+NodePlugin.RULES_DIR = path.resolve(__dirname, 'tools', 'eslint-rules');
+
+const ModuleFindPath = Module._findPath;
+const hacks = ['eslint-plugin-node', 'eslint-plugin-markdown', 'babel-eslint'];
```

Comment:
Maybe use `eslint-plugin-node-core` as the namespace since [`eslint-plugin-node`](https://www.npmjs.com/package/eslint-plugin-node) is already taken and seems quite popular? I have put a placeholder in `eslint-plugin-node-core` to [export this rules out](https://github.com/nodejs/admin/issues/55)

Reply:
if we do that then all our custom rules become `node-core/rule-name` which is why i chose to make a custom `eslint-plugin-node` instead of using `eslint-plugin-rulesdir`

Reply:
Also, I am not an expert on eslint but I think if we publish `eslint-plugin-node-core` to npm and install them globally, the editors are going to work as well? (The error documentation rules would be a bit tricky since they need to read `doc/api/errors.md` though)

Reply:
if installed globally and run with global eslint install then yes it would work 👍